### PR TITLE
add video extension

### PR DESCRIPTION
**Pull request by lucienicolier**
https://github.com/sphinx-contrib/video

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ sphinx==1.8.5
 Pygments>=2.0.1
 sphinx_bootstrap_theme
 transifex-client
+pbr
```
Comment (signedav) [inline requirements.txt:5]: I don't have much experience with sphinx but according to this https://github.com/opengisch/QField-docs/pull/100/checks?check_run_id=1509285590#step:7:289 it's not properly installed. I expected that there could be done a `pip install sphinxcontrib.video` but apparently it's not on pypi, so you could try `pip install git+https://github.com/sphinx-contrib/video.git` according to https://github.com/sphinx-contrib/video/issues/2

**Comment by lucienicolier** [comment]
@signedav I did add the pip extension, is that right?

**Comment by signedav** [comment]
I expected to have it in the requirements.txt but shouldn't make a difference.
It's running now through the workflow... But still not working?

Hm. Maybe @mbernasocchi has an idea.

**Comment by lucienicolier** [comment]
@mbernasocchi I added an example with the search video.

**Review by mbernasocchi — APPROVED** [review]

**Comment by m-kuhn** [comment]
There seems to be a problem with this pull request (see the chat on Ninjas)

- The .webm file is not published (at least not to the expected location in `_images`).
- This pull request removed two videos (search and go to coordinates) and replaced the go to coordinates video with the search video

@mbernasocchi can you have a look? Thanks

**Pull request merged**